core/impl/src/test/java/org/eclipse/sensinact/core/twin/impl/SensinactTwinTest.java

```diff
@@ -113,12 +113,11 @@ void start() throws IOException {
                 .withAction(List.of(new SimpleEntry<>("foo", String.class), new SimpleEntry<>("bar", Instant.class)))
                 .buildAll();
 
-        manager.createModel(TEST_MODEL_WITH_METADATA).withService(TEST_SERVICE)
-                    .withResource(TEST_RESOURCE).withType(Integer.class).withDefaultMetadata(Map.of("foo", "bar")).build()
-                    .withResource(TEST_ACTION_RESOURCE).withType(Double.class)
-                        .withAction(List.of(new SimpleEntry<>("foo", String.class), new SimpleEntry<>("bar", Instant.class)))
-                        .withDefaultMetadata(Map.of("fizz", "buzz"))
-                .buildAll();
+        manager.createModel(TEST_MODEL_WITH_METADATA).withService(TEST_SERVICE).withResource(TEST_RESOURCE)
```

Comment:
Please try to avoid whitespace changes in unrelated files as it makes it look like there may have been behaviour changes!